docs/configuration.md

```diff
@@ -536,8 +536,14 @@ The idea here is to place the sell order early, to be ahead in the queue.
 
 A fixed slot (mirroring `bid_strategy.order_book_top`) can be defined by setting `ask_strategy.order_book_min` and `ask_strategy.order_book_max` to the same number.
 
-!!! Warning "Orderbook and stoploss_on_exchange"
-    Using `ask_strategy.order_book_max` higher than 1 may increase the risk, since an eventual [stoploss on exchange](#understand-order_types) will be needed to be cancelled as soon as the order is placed.
+!!! Warning "Order_book_max > 1 - increased Risk!"
+    Using `ask_strategy.order_book_max` higher than 1 will increase the risk, since an eventual [stoploss on exchange](#understand-order_types) will be needed to be cancelled as soon as the order is placed.
```

Comment:
"increase the risk" of what? risk of appearance of unfilled order hanging till expired?

Reply:
increased risk as (for example) the stoploss-on-exchange is cancelled way too early - so if the market turns on you, it'll go below your stoploss.